docs/docs/api-access-rules.md

```diff
@@ -234,6 +234,27 @@ authenticators:
 }
 ```
 
+## Upstream Transport Configuration
+
```

Comment:
Probably makes sense here to document that this is supported both for proxy upstreams as well as for all authenticators, authorizers, ... that do HTTP?